glib/src/object.rs

```diff
@@ -3500,23 +3500,23 @@ impl<T: ObjectType> From<WeakRef<T>> for SendWeakRef<T> {
 unsafe impl<T: ObjectType> Sync for SendWeakRef<T> {}
 unsafe impl<T: ObjectType> Send for SendWeakRef<T> {}
 
-type TransformFn =
-    Option<Box<dyn Fn(&crate::Binding, &Value) -> Option<Value> + Send + Sync + 'static>>;
+type TransformFn<'b> =
+    Option<Box<dyn Fn(&'b crate::Binding, &'b Value) -> Option<Value> + Send + Sync + 'static>>;
 
 // rustdoc-stripper-ignore-next
 /// Builder for object property bindings.
 #[must_use = "The builder must be built to be used"]
-pub struct BindingBuilder<'a> {
+pub struct BindingBuilder<'a, 'f, 't> {
     source: &'a ObjectRef,
     source_property: &'a str,
     target: &'a ObjectRef,
     target_property: &'a str,
     flags: crate::BindingFlags,
-    transform_to: TransformFn,
-    transform_from: TransformFn,
+    transform_from: TransformFn<'f>,
+    transform_to: TransformFn<'t>,
```

Comment:
Why did you change the order? :)

Reply:
So the signature for the Builder uses `<'a, 'f, 't>` instead of `<'a, 't, 'f>`. :)

Reply:
OK, so no deeper reason :) Thanks